### DISCUSSION TITLE
Uninstall Analyst Console

### DISCUSSION THREAD

**Discussion by zaphode** [Security-Onion-Solutions/securityonion#5304] [General]
Is there a way to uninstall the Analyst console once you have run so-analyst-install (or rerun it to detect more screen resolutions once video memory is added)

**Comment by dougburks** [comment]
You shouldn't need to uninstall the analyst console to change the screen resolution. You should be able to do it from the System Settings window. If you are doing this in a virtual machine, then you may need to install the virtualization tools for your virtualization platform.
